torch/csrc/jit/serialization/import.h

```diff
@@ -99,6 +99,12 @@ TORCH_API Module load(
     c10::optional<c10::Device> device = c10::nullopt,
     bool load_debug_files = true);
```

Comment:
not particularly opinionated about this but what do you think about just adding this as a parameter in the version of jit::load on line 97, and maybe the same in line 81?